src/0.9.0/immudb/README.md

```diff
@@ -400,19 +423,18 @@ immuclient_audit_result_per_server{server_address="127.0.0.1:3322",server_id="br
 # TYPE immuclient_audit_run_at_per_server gauge
 immuclient_audit_run_at_per_server{server_address="127.0.0.1:3322",server_id="br8eugq036tfln0ct6o0"} 1.5907565337454605e+09
 ```
+It's possible to provide the public key to verify that the signature of immudb.
+```bash
+./immuclient audit-mode --audit-databases defaultdb --audit-password immudb --audit-username immudb  --server-signing-pub-key ./test/signer/ec1.pub
+```
+If the server public signing key is being provided to the auditor each not signed state will trigger an error.
```

Comment:
I would change this to "If the server public signing key is being provided to the auditor, each unsigned state will trigger an error."